### PR TITLE
Add class to calculate the classification and regression metrics

### PR DESCRIPTION
**Pull request by pan000987**
Hi mate,

I wrote the two python classes to **calculate the classification and regression metrics**.

The _Dependent Library i_s **Numpy**.

For both of the classes, _the type of the input variables_ is **list or np.ndarray**; and _the input variables_ are **actual_y and predicted_y**.

The metrics for _classification_ are **accuracy, precision, recall and F1-score**.
The metrics for _regression_ are **MSE, RMSE, MAE, MAPE**.

The test cases are under each class.

If something goes wrong, please hesitate to contact me. Cheers mate.

### CODE REVIEW

**Comment by fsx950223** [comment]
Why don't you use tf.keras.metrics or sklearn?
